config/ldp/modes/default.json

```diff
@@ -7,7 +7,24 @@
       "@type": "WaterfallHandler",
       "handlers": [
         { "@type": "MethodModesExtractor" },
```

Comment:
Maybe some comments to the extent of "extract permissions based on the method" / "extract permissions based on the request body".